### DISCUSSION TITLE
Lettering Color Options

### DISCUSSION THREAD

**Discussion by 19msullivan** [inkstitch/inkstitch#1933] [Q&A]
Hi! 

I've been loving the pre-made lettering in inkstitch, but I was wondering if there is a way to change the color from black? Specifically, I have been using Espresso KOR. If not that is fine, I just want to be sure I am not overlooking a feature:)

Thanks!

**Comment by tyrosinase** [comment]
The single-color lettering is pretty easy to change: just select it and change the stroke color (be careful not to set a fill by mistake).

Multi-color alphabets are only a little harder - select one segment of a color (shift-click to not select the whole group), then right-click and Select Same on the stroke color, then change it. You have to be careful that doesn't also select other elements of your design, but you can re-color the lettering in a different document and then copy it over if necessary.

**Comment by 19msullivan** [comment]
Oh I see! When I select the object, and then go to the "stroke" tab, it says "paint is undefined" so I was worried I wouldn't be able to modify it. It worked though! Thank you so much for your help!! :)
